runtime/context.go

```diff
@@ -32,6 +33,16 @@ const (
 	endpointKey        = contextFieldKey("endpoint")
 	requestUUIDKey     = contextFieldKey("requestUUID")
 	routingDelegateKey = contextFieldKey("rd")
+	requestLogFields   = contextFieldKey("requestLogFields")
+)
+
+const (
+	logFieldRequestMethod       = "method"
```

Comment:
How should we differentiate the field between client and endpoint?

Reply:
I'm preserving these log fields as they were before, open to better names. `endpoint-http-method`, `endpoint-url`, etc. seem better to me. Thoughts?

Reply:
That works, we should also think about how these can be customizable so that user can define log fields.